bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -393,35 +231,34 @@ logging.info("SNR and bestNR maxima calculated.")
 
 # Output details of loudest offsouce triggers, sorted by BestNR
 offsource_trigs = []
-sorted_trigs = ppu.sort_trigs(trial_dict, trigs, slide_dict, segment_dict)
+sorted_trigs = ppu.sort_trigs(trial_dict, trig_data, slide_dict, segment_dict)
 for slide_id in slide_dict:
-    offsource_trigs.extend(zip(trig_bestnr[slide_id], sorted_trigs[slide_id]))
+    offsource_trigs.extend(zip(trig_bestnr[slide_id],
+                               get_trig_dictionary(sorted_trigs[slide_id])))
```

Comment:
I would leave this alone and carry around the `event_id`s of triggers. Then you can get what you need directly from the file.